plugins/analytics-module-ga/src/apis/implementations/AnalyticsApi/GoogleAnalytics.ts

```diff
@@ -157,11 +162,15 @@ export class GoogleAnalytics implements AnalyticsApi {
    * pageview and the rest as custom events. All custom dimensions/metrics are
    * applied as they should be (set on pageview, merged object on events).
    */
-  captureEvent(event: AnalyticsEvent) {
+  captureEvent(event: AnalyticsEvent | NewAnalyticsEvent) {
     const { context, action, subject, value, attributes } = event;
     const customMetadata = this.getCustomDimensionMetrics(context, attributes);
 
-    if (action === 'navigate' && context.extension === 'App') {
+    const extensionId = context.extensionId || context.extension;
+    const category = extensionId ? String(extensionId) : 'app';
```

Comment:
As discussed, switching the default to `'App'` just in case, but in practice we never really expect that to be reached. Just being extra careful not to break existing usage. This should disappear once this only supports the new analytics API anyway.

Reply:
Fixed 🙌🏻